tests/cases/corner_cases.cddl

```diff
@@ -152,3 +152,9 @@ Prelude = [
 	float32-64,
 	null,
 ]
+
+CBORBstr = bstr .cbor ([
```

Comment:
We support comments here right? Would it be an idea to have a comment here (not to the other examples) why this is considered a corner-case?

Reply:
Added